packages/zowe-explorer/src/Profiles.ts

```diff
@@ -482,10 +482,24 @@ export class Profiles extends ProfilesCache {
                             if (!Number.isNaN(Number(updValue))) {
                                 updSchemaValues[value] = Number(updValue);
```

Comment:
If input is an empty string (i.e. the user hit enter), `Number("") === 0`
This results in the profile still being saved with `responseTimeout: 0`.

We could check for `updValue !== "" && !Number.isNaN(Number(updValue)` but that leads to another problem. (see next comment)

This comments applies to both Creating a new Connection (`createNewConnection`) and Editing a session (`editSession`)